Cargo.toml

```diff
@@ -15,4 +15,4 @@ keywords = ["data-structures"]
 readme = "README.md"
 
 [dependencies]
-linked-hash-map = "*"
+linked-hash-map = { git = "https://github.com/contain-rs/linked-hash-map" }
```

Comment:
I'm not in favor of changing this to a dependency on the repo. I'll publish a new version of `linked-hash-map` shortly.